site/realm/data_sources/mongodb-atlas/aiidprod/checklists/rules.json

```diff
@@ -0,0 +1,26 @@
+{
+    "collection": "checklists",
+    "database": "aiidprod",
+    "roles": [
+        {
+            "name": "role is admin",
+            "apply_when": {
+                "%%user.custom_data.roles": "admin"
+            },
+            "read": true,
+            "write": true,
+            "insert": true,
+            "delete": true,
+            "search": true
+        },
+        {
+            "name": "default",
+            "apply_when": {},
+            "read": true,
+            "write": true,
+            "insert": true,
+            "delete": true,
+            "search": true
+        }
```

Comment:
Yes, right now there's no authentication https://github.com/responsible-ai-collaborative/aiid/issues/2300. I think this can come after merging but before release.